src/main/java/io/vertx/core/VertxOptions.java

```diff
@@ -412,7 +412,7 @@ public int getQuorumSize() {
    * @return a reference to this, so the API can be used fluently
    */
   public VertxOptions setQuorumSize(int quorumSize) {
-    if (quorumSize < 1) {
+    if (quorumSize <= 0) {
```

Comment:
```suggestion
    if (quorumSize < 1) {
```

Reply:
@tsegismont If quorumSize is less than 1, then Exception message is incorrect here.
                           `throw new IllegalArgumentException("quorumSize should be >= 1");` 
 This will break if we pass 1 as message is expecting it can be 1.

Reply:
Oh yeah, sorry :blush:

Reply:
Actually no, the test was fine. If `quorumSize` is set to 0 or less, the test fails and the message is printed.
If you give 1, then the test passes and the option field is set to 1.